### ISSUE TITLE
Suggestion: Ability to self-initiate a call from the Officer MDT

### ISSUE THREAD

**Issue by jmoney330** [StormlightTech/OpenCAD-php#77]
I'd like to see the ability to self-initiate a call on the MDT side of things. This would come in handy should an officer want to a simple action such as a premises check or maybe speed enforcement in a particular area. This would also be useful if dispatch is very busy and the officer wants to mark himself down so dispatch knows where he is but does not want to tie up the dispatcher by calling out something so simple. 

Basically it would just be like a dispatcher adding the call, however the only difference would be the officer does it and after the call is created the officer would automatically be assigned and marked on scene of the call.
